compiler/statements.go

```diff
@@ -39,10 +44,29 @@ func DoMetaCommand(buffer InputBuffer) MetaCommandResult {
 
 func PrepareStatement(buffer InputBuffer, statement *Statement) PrepareResult {
 	if len(buffer.Buffer) > 6 {
-		if buffer.Buffer[:6] == "insert" {
-		statement.Type = StatementInsert
-		return PrepareSuccess
-	}
+		// bufferArguments contains a slice of arguments in buffer.Buffer separated by a space
```

Comment:
Let's take comments off the master branch. We could make such comments in the Slack :)